app/src/main/java/backgammon/DoublingCube.java

```diff
@@ -17,10 +17,19 @@ public void setCubeValue(int x){
     this.cubeValue = x;
   }
 
+  //doubles the cube value
   public void doubleCubeValue(){
     this.cubeValue = this.cubeValue * 2;
   }
 
+  //checks if a player is eligible to use the doubling cube (right now prints yay or nay)
+  public void cubeCheck(Boolean player){
+    if(player && this.cubeValue < 64)
+      System.out.println("yay");
+    else
+      System.out.println("nay");
+
+  }
```

Comment:
Funny.
Add that check in the `doubleStakes` method instead.
When you move the check to `doubleStakes` remove the print statements because you'll be using it in `doubleStakes` to cap the max value at 64 anyways.
Remove this method or I will.